.github/ISSUE_TEMPLATE/new-species-issue-template.md

```diff
@@ -0,0 +1,61 @@
+---
+name: New species
+about: Adding a new species to the stdpopsim catalogue.
+title:
+labels: enhancement
+assignees: ''
+
+---
+
+- **Name:** scientific name of the species
+- **Common name:**
+
+**Description:** any other discussion or useful background? (optional)
+
+*Note: Maybe you don't know a lot of the things below - but, that's OK!
+The criteria are that (a) the choice is reasonable, and (b) the rationale is clear.
+What we want is a "reasonable estimate" (or, "best guess") - if you were to use a number in a paper,
+what would you use and how would you justify it to your collaborators?
+For instance, for "generation time", best would be to cite a paper that actually measures
+or estimates generation time. But most species don't have this;
+next best would be to give a number used in the literature, and provide a citation that used it
+(and hopefully the publication gives a justification also).
+Lacking this, you might provide a number from a related species (and a citation for this).*
+
+*Here is the checklist of things that we need to add a species to the catalogue.
+Each thing should be provided, with a justification (maybe short) and a citation.*
+
+**Demographic information:**
+
+- [] generation time in years
+- [] "default" population size
+
+
+**Chromosome structure:**
+
+- [] list of chromosomes with *name* and *length* (in bp)
+
+
+**Recombination rates:**
+
+- [] genetic map (as a .csv) of recombination rates **(optional)**
```

Comment:
Or Hapmap? Don't want to people to convert to CSV if it's already in the right format.